src_c/surface.c

```diff
@@ -711,6 +711,18 @@ surface_init(pgSurfaceObject *self, PyObject *args, PyObject *kwds)
     if (!(flags & PGS_SRCALPHA)) {
         /* We ignore the error if any. */
         SDL_SetSurfaceBlendMode(surface, SDL_BLENDMODE_NONE);
+
+        /* When the display format has a full alpha channel (MacOS right now),
+         * Surfaces may be created with an unreqested alpha channel, which
+         * could cause issues.
+         * pygame Surfaces are supposed to be (0, 0, 0, 255) by default.
+         * This is a simple fix to fill it with (0, 0, 0, 255) if necessary.
+         * See Github issue: https://github.com/pygame/pygame/issues/1395
+         */
+        if (Amask != 0) {
```

Comment:
I wonder if we can avoid this on other platforms that don't need it?

Reply:
I expect it's not a performance issue on small surfaces, because the memory is probably already in cache. But for larger ones this could be a significant slow-down.

Reply:
It is avoided on platforms that don't need it. The only time it fires is if Amask is some number, but SRCALPHA wasn't requested. That only happens if the preferred display pixel format has alpha, which we've determined only happens on Mac.

Reply:
Cool. Thanks for the explanation.